docs/tsconfig.json

```diff
@@ -19,5 +19,5 @@
     "target": "esnext",
     "resolveJsonModule": true
   },
-  "include": ["../docs/**/*.ts*"]
```

Comment:
I think you must use .tsx for store. It should not be complicated. ASFAIK there is should not be any type definitions

Reply:
Following prev comment. Just put it inside the example